app/serializers/api/application_serializer.rb

```diff
@@ -0,0 +1,57 @@
+module API
+  class ApplicationSerializer < Blueprinter::Base
+    identifier :ecf_id, name: :id
+
+    class AttributesSerializer < Blueprinter::Base
```

Comment:
could we add the type to the serialiser `"npq_application"`?

Reply:
could we also order the fields similar to before except for the timestamps, happy for them to go last

Reply:
I've added the `type` and tried to copy the ordering in ECF but a few attributes are defined twice in ECF so the ordering isn't quite clear, let me know if I haven't got it right.